### PR TITLE
Redesign of landing page after login

### PR DESCRIPTION
**Pull request by anitacaron**

### CODE REVIEW

File: app/templates/index.hbs [hunk 1]
```diff
@@ -1,44 +1,115 @@
...
+
+      <AuHeading @level="2" @skin="3" class="au-u-margin-bottom">
+        Beschikbare modules
+      </AuHeading>
+
+      <ul class="au-o-grid au-o-grid--small">
+        <li class="au-o-grid__item au-u-1-2@medium au-u-1-3@large">
+          <AuCard @flex={{true}} as |c|>
+            <c.header @badgeSkin="brand" @badgeIcon="sitemap">
+              <AuHeading @level="2" @skin="4">
```
Comment (Windvis) [inline app/templates/index.hbs:45]: I think these need to be level 3 since there is already a h2 parent, right?

**Review by Windvis — CHANGES_REQUESTED** [review]
Great job! Only some small nitpicky things again.

File: app/templates/index.hbs [hunk 2]
```diff
@@ -1,44 +1,115 @@
...
+    >
+      <AuHeading @skin="2">
+        Lokale Financiën - Jan Janssens
+      </AuHeading>
+
+      <AuHeading @level="2" @skin="3" class="au-u-margin-bottom">
+        Beschikbare modules
+      </AuHeading>
+
+      <ul class="au-o-grid au-o-grid--small">
```
Comment (Windvis) [inline app/templates/index.hbs:41]: `au-o-grid--small` should be dropped, the white space is larger in the design.

File: app/templates/index.hbs [hunk 3]
```diff
@@ -1,44 +1,115 @@
...
+        <a class="au-c-link au-c-link--secondary au-c-link--icon" href="" target="_blank" rel="noopener noreferrer">
+          <AuIcon @icon="manual" @alignment="left" />
+          Handleiding
+        </a>
+      </div>
     </div>
+  </div>
+  <div class="au-c-main-container__content au-c-main-container__content--block">
+    <div
+      class="au-o-box au-o-flow au-o-flow--large"
```
[the remark this answers is not in the document]
Comment (anitacaron) [reply app/templates/index.hbs:31]: So, in the end, it won't have space between the headers?

[the remark this answers is not in the document]
Comment (Windvis) [reply app/templates/index.hbs:31]: There will be if you add `au-u-margin-bottom-large` to the h1.

File: app/templates/index.hbs (hunk 1 above)
Comment (anitacaron) [reply app/templates/index.hbs:45]: Same result.

Comment (Windvis) [reply app/templates/index.hbs:45]: Not sure what you mean with that :smile:

File: app/templates/index.hbs (hunk 3 above)
[the remark this answers is not in the document]
Comment (anitacaron) [reply app/templates/index.hbs:31]: Got it.

File: app/templates/index.hbs (hunk 1 above)
Comment (Windvis) [reply app/templates/index.hbs]: Ah yea, the `@level` only changes the element, not the styling.

**Review by Windvis — APPROVED** [review]
Nice!

**Pull request merged**